katello.dot

```diff
@@ -27,27 +27,27 @@ node "server" {
     [Foreman Discovery] as foreman_discovery
     [Foreman SCAP] as foreman_scap
 
-    foreman_tasks -down-> dynflow
-    katello -> foreman
-    katello -> foreman_tasks
-    katello -> foreman_docker
+    foreman_tasks -down..> dynflow
+    katello ..> foreman
+    katello ..> foreman_tasks
+    katello ..> foreman_docker
 
-    foreman_discovery -> foreman
-    foreman_docker -> foreman
-    foreman_scap -> foreman
-    foreman_tasks -> foreman
+    foreman_discovery ..> foreman
+    foreman_docker ..> foreman
+    foreman_scap ..> foreman
+    foreman_tasks ..> foreman
 
     dynflow ..> postgres
   }
 
   server_apache ..> foreman
 
-  foreman -down..> postgres : localhost
-  candlepin -down..> postgres: localhost
+  foreman -[#red]down-> postgres : localhost
+  candlepin -[#red]down-> postgres: localhost
 
-  katello -down..> candlepin : localhost:8443
+  katello -[#red]down-> candlepin : localhost:8443
```

Comment:
This shouldn't be red should it?  looks like its using https to me.  I would think it would be cyan.